src/components/MapView.jsx

```diff
@@ -12,20 +15,28 @@ import 'leaflet/dist/leaflet.css'
 //     shadowUrl: markerShadow
 // })
 
+const ChangeView = ({ center }) => {
+    const map = useMap();
+    useEffect(() => {
+        map.setView(center);
+    }, [center, map]);
 
+    return null;
+}
 const MapView = ({ clientes, sectores, center }) => {
     return (
         <MapContainer
-            center={center}
-            zoom={10}
+            center={[departments.Cochabamba.lat, departments.Cochabamba.lng]}
```

Comment:
This is a static default value for the center, and is already defined in App.jsx component.   const [mapCenter, setMapCenter] = useState({ lat: -16.290154, lng: -63.588653 }); here.